packages/i18n/src/prebuild.mjs

```diff
@@ -34,14 +34,19 @@ const getTranslationFileList = async () => {
   return files.sort()
 }
 
+/** extracts the locale from the file name */
+const localeFromFileName = (fileName) => {
+  const split1 = fileName.split(`${path.sep}locales${path.sep}`).pop()
+  return split1.split(`${path.sep}`).shift()
+}
+
 /*
  * Figures out the list of locales from the list of files
  * (by checking how many version of aaron.yaml exist)
  */
```

Comment:
Should we perhaps keep a list of (supported) locales in the repo's config folder? We might need this in more place than one 🤔

Reply:
wow, I was calling `return` instead of `continue` when checking locales, so that's why it was breaking with a filter

Reply:
> My question is what the use case is for not simply using all the locales that we have?

Well people are currently working on the Ukranian translation. It is 3% done, and we want it to go up to a higher number, and at least have the signup/login flow and so on translated.
When that's done, we'll probably do some sort of *we haz ukranian now* announcement. But until that time, it should be kept out of the build.

For what it's worth, adding a new language is an exceedingly rare event.